staging/src/k8s.io/dynamic-resource-allocation/structured/allocator_test.go

```diff
@@ -240,13 +240,15 @@ func slice(name string, nodeSelection any, pool, driver string, devices ...resou
 	return slice
 }
 
-func deviceAllocationResult(request, driver, pool, device string) resourceapi.DeviceRequestAllocationResult {
-	return resourceapi.DeviceRequestAllocationResult{
+func deviceAllocationResult(request, driver, pool, device string, adminAccess bool) resourceapi.DeviceRequestAllocationResult {
```

Comment:
To make this a bit easier in the future, maybe you could define a constant for adminAccess so that it is clear what these parameter is during the test code.

Usually for API fields, we discourage bools as they are not very that descriptive but I think this is probably fine as a bool. But we sho

Reply:
Should what?